src/components/policy/src/policy/src/cache_manager.cc

```diff
@@ -205,8 +205,20 @@ bool CacheManager::ApplyUpdate(const policy_table::Table& update_pt) {
     }
   }
 
+
+  for (policy_table::ApplicationPolicies::iterator it =
+        pt_->policy_table.app_policies.begin();
+        pt_->policy_table.app_policies.end() != it;
+        ++it) {
+    if (IsDefaultPolicy(it->first)) {
+      SetDefaultPolicy(it->first);
+    }
+  }
+
   pt_->policy_table.module_config.country_consent_passengersRC =
       update_pt.policy_table.module_config.country_consent_passengersRC;
+  pt_->policy_table.module_config.equipment =
```

Comment:
I can ask you about it too.
I have just found new requirement. It say we need update this section but this feature is in backlog now.
I can't find this requirement in old list. But I remember that our the requirements manager told me that equipment is in preloaded pt only.
I have shared my doubt only, please clarify this.